src/main/java/org/isf/pricesothers/gui/PricesOthersEdit.java

```diff
@@ -46,51 +44,51 @@
 import org.isf.utils.exception.gui.OHServiceExceptionUtil;
 import org.isf.utils.jobjects.MessageDialog;
 import org.isf.utils.jobjects.VoLimitedTextField;
+import org.isf.utils.layout.SpringUtilities;
 
 public class PricesOthersEdit extends JDialog {
 	
-	private EventListenerList OthersListeners = new EventListenerList();
+	private EventListenerList othersListeners = new EventListenerList();
```

Comment:
Might we refactor into `PricesOthersListeners` ?

Reply:
Yes we can.  Note it should start with a lowercase character.  I went ahead and changed method names to be more clear.

Reply:
sure, thanks!